lib/AST/GenericSignatureBuilder.cpp

```diff
@@ -2316,6 +2318,7 @@ Type EquivalenceClass::getTypeInContext(GenericSignatureBuilder &builder,
 
 void EquivalenceClass::dump(llvm::raw_ostream &out,
                             GenericSignatureBuilder *builder) const {
+#if defined(SWIFT_PERMIT_UNDEFINED_BEHAVIOR)
```

Comment:
Rather than do this `SWIFT_PERMIT_UNDEFINED_BEHAVIOR` dance that makes `dump()` useless, let's just outright remove the GraphViz output for equivalence classes. It's actually not that useful for debugging right now, and we can bring it back properly when we need it.

Reply:
Okay, that sounds good to me.  I'll just remove that part.